mne/viz/utils.py

```diff
@@ -1691,9 +1696,17 @@ def _compute_scalings(scalings, inst):
                                  % (key, value))
             continue
         this_data = data[ch_types[key]]
-        scale_factor = np.percentile(this_data.ravel(), [0.5, 99.5])
-        scale_factor = np.max(np.abs(scale_factor))
-        scalings[key] = scale_factor
+        if remove_dc and (this_data.shape[1] / inst.info["sfreq"] >= 10):
```

Comment:
Why the check for `>= 10` sec of data? Shouldn't it at least be `>= duration` (plot duration) sec?

Reply:
Ahh I see why it's necessary, because you use reshape. So just needs to be changed to be `duration` instead of `10`.

Reply:
Initially I thought I'd calculate this on 10s windows, independently from whatever `duration` is. But it probably makes more sense to always use `duration`.

Reply:
I see why I used 10s - I didn't want to add another parameter. I'd leave the hard-coded 10s in, but if you insist I'll add duration as a new parameter.

Reply:
No don't use a new parameter. Use the existing parameter that determines the interval over which DC removal is actually done, namely the plot duration / window being shown.

Reply:
(I mean "user-facing parameter". If you mean "parameter in this function" I see no reason not to add a new `kwarg` since it's private anyway.)

Reply:
Yes, I meant parameter in this function. I've added a new one.